File: Pattern/5.py
# * * * * *
# * * * * 
# * * *
# * *
# *


n = int(input("Enter the number: "))

for i in range(n, 0, -1): #5 and 1st row will be like 5 and next row will be 5-1 untill line reaches 5
    for j in range(i): # 5 coulumns or i coloums
        print("*", end=" ")
    print("\r")

File: Patterns in JS/5.js
// 1
// 2 2
// 3 3 3
// 4 4 4 4
// 5 5 5 5 5

function five(n) {
  for (let i = 0; i < n; i++) {
    let row = "";
    for (let j = 0; j < i + 1; j++) {
      row += i + 1 + " ";
    }
    console.log(row);
  }
}

five(5);
